src/turnkeyml/cli/report.py

```diff
@@ -78,22 +79,17 @@ def summary_spreadsheets(args) -> None:
 
                         # Copy the build-specific stats
                         for key, value in build.items():
-                            # Break each value in "completed build stages" into its own column
-                            # to make analysis easier
-                            if key == fs.Keys.COMPLETED_BUILD_STAGES:
-                                for subkey, subvalue in value.items():
-                                    evaluation_stats[subkey] = subvalue
-
-                            # If a build or benchmark is still marked as "running" at
+                            # If a build or benchmark is still marked as "incomplete" at
                             # reporting time, it
                             # must have been killed by a time out, out-of-memory (OOM), or some
                             # other uncaught exception
                             if (
                                 key == fs.Keys.BUILD_STATUS or fs.Keys.BENCHMARK_STATUS
-                            ) and value == fs.FunctionStatus.RUNNING:
-                                value = fs.FunctionStatus.KILLED
+                            ) and value == bd.FunctionStatus.INCOMPLETE.value:
```

Comment:
Should

```
                        if (
                                key == fs.Keys.BUILD_STATUS or fs.Keys.BENCHMARK_STATUS
                            ) and value == bd.FunctionStatus.INCOMPLETE.value:
```
Simply be
```
                        if (value == bd.FunctionStatus.INCOMPLETE.value):
```
?

I thought `INCOMPLETE` should never appear in the report

ERROR means INCOMPLETE_WITH_ERROR
KILLED means INCOMPLETE_WITHOUT_ERROR

Reply:
wont fix: that would be risky since stages and other functions can also be incomplete. filtering by key prevents the logic from being accidently applied to other functions

Reply:
My suggestion is to avoid overlapping categories in the report. `INCOMPLETE` overlaps in terms of meaning with both `ERROR` and `KILLED`.
* Non-overlapping example: `(NOT_STARTED, SUCCESSFUL, ERROR, KILLED)` [Type 1]
* Non-overlapping example: `(NOT_STARTED, SUCCESSFUL, INCOMPLETE)` [Type 2]
* Overlapping Example: `(NOT_STARTED, SUCCESSFUL, ERROR, INCOMPLETE)`

It looks like `build` and `benchmark` have report with results of `Type 1`. Can we also make the same for `stages` or at least make them `Type 2`? As long as those don't have overlapping categories the results are easy to understand.

Alternative (not sure if we want to go there): Replace `incomplete` with `incomplete_without_error` , error with `incomplete_with_error`, and remove killed (already part of `incomplete_without_error`). This way there is no need for post-processing anything in `report.py`

Reply:
TLDR: please replace INCOMPLETE with KILLED in stages as well :)

Reply:
Implemented!